zebra-node-services/src/mempool.rs

```diff
@@ -0,0 +1,122 @@
+//! The Zebra mempool.
+//!
+//! A service that manages known unmined Zcash transactions.
+
+use std::collections::HashSet;
+
+use tower::BoxError;
```

Comment:
Suggestion (1/4): Can we copy the definition of `zebrad::BoxError`, and drop the `tower` dependency?

https://github.com/ZcashFoundation/zebra/blob/e75fe2c42faab4ae6f7780e07879a54bafe73efd/zebrad/src/lib.rs#L27-L32

Reply:
Done.